src/Microsoft.ML.Transforms/NormalizerCatalog.cs

```diff
@@ -1,4 +1,5 @@
-﻿using Microsoft.ML.Data;
+﻿using System.Linq;
```

Comment:
While you are in here - can you add a copyright to this file?

Reply:
totally, good catch.

---
In reply to: [271425802](https://github.com/dotnet/machinelearning/pull/3172#discussion_r271425802) [](ancestors = 271425802)